raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -106,4 +100,23 @@ library ChannelManagerLibrary {
 
         return size > 0;
     }
+
+    /// @notice Get the hash of the two addresses
+    /// @param address_one address of one party
+    /// @param address_two of the other party
+    /// @return The sha3 hash of both parties sorted by size of address
```

Comment:
maybe also change the comment here, to denote it's the keccak256 sha-3 function?

Reply:
changed all occurrences, including additional ones in the channel manager and token contracts.